JiraPS.build.ps1

```diff
@@ -194,6 +194,13 @@ task UpdateManifest GetNextVersion, {
     if ($ModuleAlias) {
         BuildHelpers\Update-Metadata -Path "$env:BHBuildOutput/$env:BHProjectName/$env:BHProjectName.psd1" -PropertyName AliasesToExport -Value @($ModuleAlias.Name)
     }
+
+    $Prerelease = ''
+    if ("$env:BHBranchName" -notin @('master','main')) {
```

Comment:
if you are checking the branch for both `master` and `main` -- which I am not opposed to -- should we use both values in the workflows too? (.github/workflows/build_and_test.yml#6)